src/redux/slices/accountSlice.ts

```diff
@@ -26,13 +31,27 @@ const sessionSlice = createSlice({
       ...state,
       accounts: payload,
     }),
-    unLoadAccount: () => ({
-      ...intialState,
+    unLoadAccount: (state) => ({
+      ...state,
+      accounts: intialState.accounts,
+    }),
+    loadUserInfo: (state, { payload }: PayloadAction<userInfo>) => ({
+      ...state,
+      user: {
+        id: payload.id,
+        name: payload.name,
+        email: payload.email,
+      },
```

Comment:
Si el payload contiene solamente estos atributos usaría esta sintáxis para simplificar
```suggestion
      user: { ... payload },
```